src/createStore.js

```diff
@@ -136,12 +136,12 @@ export default function createStore(reducer, preloadedState, enhancer) {
     nextListeners.push(listener)
 
     return function unsubscribe() {
-      if (!isSubscribed) {
+      if (! isSubscribed) {
         return
       }
 
       if (isDispatching) {
-        throw new Error(
+        throw new Error (
```

Comment:
format: style mismatch:
```suggestion
        throw new Error(
```

`e754d017` ␣ at column 24 should be removed.